src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -82,6 +84,15 @@
 
 # ACS
 
+def transform_acs_list(result):
+    transformed = []
+    for r in result:
+        res = OrderedDict([('Name', r['name']), ('ResourceGroup', r['resourceGroup']), \
+            ('Orchestrator', r['orchestratorProfile']['orchestratorType']),
```

Comment:
If orchestratorProfile is ever null, this will fail and you'll get the "Table format not supported" message. If this parameter is guaranteed to never be null then it will be fine, but I don't know enough about the domain to make that call.